src/main/java/PersonalBudget/business/user/domain/model/UserEntity.java

```diff
@@ -16,35 +16,35 @@
 import org.springframework.security.core.userdetails.UserDetails;
 
 import java.util.Collection;
+import java.util.Collections;
 
 @Getter
 @Setter
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
 @Entity
-@Table(name = "users")
+@Table(name = "user_account")
```

Comment:
I found the comment on stackoverflow as a word "user" is presumably reserved: 

"I'd say that you should avoid having table names that are reserved words, with hibernate. Sure you can escape it, but it may cause problems in the future (in a query for example). So the safest way is to name the table another way"